downloads.md

```diff
@@ -200,6 +200,24 @@ You can add the following dependencies to your `pom.xml` to include Apache Flink
 The `statefun-sdk` dependency is the only one you will need to start developing applications.
 The `statefun-flink-harness` dependency includes a local execution environment that allows you to locally test your application in an IDE.
 
+### Apache Flink ML
+
+You can add the following dependencies to your `pom.xml` to include Apache Flink ML in your project.
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-ml-uber_2.12</artifactId>
+  <version>{{ site.FLINK_ML_VERSION_STABLE }}</version>
+</dependency>
+```
+
+flink-ml-uber_2.12 provides all the dependencies needed to use Flink ML. Alternatively, advanced users could use the following artifacts to import a minimal set of Flink ML dependencies for their target use-cases:
```

Comment:
It seems that flink-ml-uber_2.12 only include `flink-ml-core` and `flink-ml-iteration` [1]. Do we mean to include all other dependencies in flink-ml?

[1] https://github.com/apache/flink-ml/blob/master/flink-ml-uber/pom.xml

Reply:
Hi @zhipeng93 I think you are right, I removed dependency of `flink-ml-uber` currently and use directly the direct dependencies . We could re-enabled the `uber` jar in the next version.